yogstation/code/datums/antagonists/vampire.dm

```diff
@@ -99,6 +106,8 @@
 	to_chat(owner, "<span class='danger bold'>You are a creature of the night -- holy water, the chapel, and space will cause you to burn.</span>")
 	to_chat(owner, "<span class='notice bold'>Hit someone in the head with harm intent to start sucking their blood. However, only blood from living creatures is usable!</span>")
 	to_chat(owner, "<span class='notice bold'>Coffins will heal you.</span>")
+	if(full_objectives == FALSE)
+		to_chat(owner, "<span class='notice bold'>You are not required to respect or obey other vampires in any way.</span>")
```

Comment:
Won't this just make new vampires able to "grief" their starting vampire?
If someone is still mad at the vampire that gave them vampire powers they can just go after their creators since they'll most likely be released after they get converted.

With this flavor text, it now becomes an option IMO.

Reply:
I don't want to turn this ability into a thrall maker, since that would just turn the gamemode into another conversion antag. I could give the victim a long sleep stun after they get converted, to give the starting vampire time to get away from them in case they get violent.

Reply:
Eh not sure honestly if you go the "sleep" route the converted guy can just "go after" the starting vampire after their stun. 

Regardless a stunning sleep or a stun might be nice because it will give the converted player time to read the vampire flavor text in the first place (remember that a lot of people being converted probably have no idea how to be a vampire)